index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio Homepage</title>
+</head>
+
+<body>
+  <div id="toolbar">
+    <div id="logoPlaceholder">
+      <p id="logoText">Welcome</p>
+    </div>
+    <input id="headerButton" type="image" src="Images/Enabled.png" alt="Menu">

```

Comment:
Your `input` element that represents your menu button must be wrapped inside a `nav` element. `nav` is short for `navigation`, as the name suggests the menu will contain all the links that the page uses for `navigation`.